descopesdk/src/main/java/com/descope/sdk/Routes.kt

```diff
@@ -854,6 +854,25 @@ interface DescopeFlow {
          */
         fun start(context: Context)
```

Comment:
Would it make sense to deprecate this function now? Or otherwise disambiguate the two with something other than the name?